uniffi/src/lib.rs

```diff
@@ -196,32 +196,6 @@ pub unsafe trait ViaFfi: Sized {
     fn try_read(buf: &mut &[u8]) -> Result<Self>;
 }
 
-/// A helper function to lower a type by serializing it into a buffer.
-///
-/// For complex types were it's too fiddly or too unsafe to convert them into a special-purpose
-/// C-compatible value, you can use this helper function to implement `lower()` in terms of `write()`
-/// and pass the value as a serialized buffer of bytes.
-pub fn lower_into_buffer<T: ViaFfi>(value: T) -> RustBuffer {
```

Comment:
I guess *technically* this is a breaking change, swapping some `pub` functions for a `pub` trait, and as such should be called out in the changelog.